components/x-audio/src/redux/swipe-down-to-minimise.js

```diff
@@ -0,0 +1,31 @@
+const EXPANDED_PLAYER_HEIGHT = 360;
+const DISPLAY_EXPANDED_THRESHOLD = EXPANDED_PLAYER_HEIGHT / 2;
+
+function setPositionY (target, posY) {
+	target.style.transform = `translate3d(0, ${posY}px, 0)`;
+}
+
+export default (event, playerActions) => {
+
+	const direction = event.offsetDirection;
+	const directionDown = direction === 16;
+
+	if (!directionDown) {
+		return;
+	}
+
+	const expandedPlayer = event.target.closest("#audio-player-expanded");
```

Comment:
Should this be using the `ref` from the main component? We could pass it in.

Reply:
@benbarnett I got expanded player ref `null` when I did 
`minimised` => `expanded` => `minimised` => `expanded(the ref is null)`,
because I didn't know they create new expanded player 🙈 
I solved this by passing ref to `listenForSwipeDown` function.

Reply:
Ah, well figured out!